conjur/logic/credential_provider/keystore_credentials_provider.py

```diff
@@ -80,6 +81,16 @@ def remove_credentials(self, conjurrc):
         """
         Method for removing user credentials in the system's keyring
         """
-        logging.debug(f"Attempting to remove credentials from the '{KeystoreAdapter.get_keyring_name()}' credential store...")
-        KeystoreAdapter.delete_password(conjurrc.conjur_url)
-        logging.debug(f"Successfully removed credentials from the '{KeystoreAdapter.get_keyring_name()}' credential store...")
+        logging.debug(
+            f"Attempting to remove credentials from the '{KeystoreAdapter.get_keyring_name()}' credential store...")
+        for attr in KEYSTORE_ATTRIBUTES:
+            try:
+                KeystoreAdapter.delete_password(conjurrc.conjur_url, attr)
+            # Catches when credentials do not exist in the keyring. If the key does not exist,
+            # the user has already logged out. we still try to remove other leftovers
+            except KeyringDeletionError as err:
```

Comment:
```suggestion
            except CredentialStoreDeletionError as credential_deletion_error:
```

Reply:
need to discuss if it is ok keystoreAdapter errors will be from the form Keyring*****Error